phpcs.xml.dist

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0"?>
 <ruleset name="Custom ruleset for Liveblog">
 
-	<rule ref="WordPress-VIP"/>
+	<rule ref="WordPress-Core"/>
```

Comment:
No need to include `WordPress-Core` if including `WordPress-Extra`, as Extra already includes Core.